sources/us/ia/statewide.json

```diff
@@ -0,0 +1,32 @@
+{
+    "coverage": {
+        "country": "us",
+        "state": "ia"
+    },
+    "data": "http://data.openaddresses.io/cache/uploads/ingalls/us-ia-2020-04-29.zip",
+    "protocol": "http",
+    "compression": "zip",
+    "conform": {
+        "format": "gdb",
+        "unit": "Unit",
+        "number": [
+            "AddNum_Pre",
+            "Add_Nuber",
+            "AddNum_Suf"
+        ],
+        "street": [
+            "StN_PreMod",
+            "StN_PreDir",
+            "StN_PreTyp",
+            "StN_PreSep",
+            "StreetName",
+            "StNPosType",
+            "StN_PosDir",
+            "StN_PosMod"
+        ],
+        "city": "MSAG_Comm",
+        "district": "County",
+        "region": "State",
+        "postcode": "Post_Code"
+    }
+t
```

Comment:
Someone stole your closing brace and replaced it with a letter!

Reply:
Haha oops!